awx/ui/src/components/PromptDetail/PromptDetail.js

```diff
@@ -87,7 +87,12 @@ function omitOverrides(resource, overrides, defaultConfig) {
   return clonedResource;
 }
 
-function PromptDetail({ resource, launchConfig = {}, overrides = {} }) {
+function PromptDetail({
+  resource,
+  launchConfig = {},
+  overrides = {},
+  workflowNode,
```

Comment:
Do we need add a default value for `workflowNode`? Since PromptDetail is used in different places.